c2corg_ui/templates/outing/view.html

```diff
@@ -106,12 +106,6 @@ <h1 class="routes">
       </h1>
     </header>
 
-  % if outing.get('areas'):
-    <article class="location-static">
-      ${show_areas(outing)}
```

Comment:
Shouldn't the `show_areas` partial be removed now? Or are postits used somewhere else?

Reply:
Please look title ... WIP ;-) ... i'm currently working on it !

Reply:
but no show area it's allready use ;-)

Reply:
Mmh. Worry, didn't see it is a WIP. But why did you request my review then? (or did I invent it?)

Reply:
Now is finish ;-)